### PR TITLE
Changed code-of-conduct link in navbar

### PR DESCRIPTION
**Pull request by baimamboukar**
### Adressing #414 
**Appliedd the various modifications as suggested [here](https://github.com/open-life-science/open-life-science.github.io/issues/414#issuecomment-1274403181)**
- changed code of conduct link in navbar with the link to code-of-conduct in policies repository
- added a small sentence message in ``CODE_OF_CONDUCT.md`` as suggested [here](https://github.com/open-life-science/open-life-science.github.io/pull/446#discussion_r992081225)

### CODE REVIEW

File: _includes/default-header.html [hunk 1]
```diff
@@ -48,7 +48,7 @@
           <div class="navbar-item has-dropdown is-hoverable">
             <a class="navbar-link" href="/about">Policies</a>
             <div class="navbar-dropdown"> 
-              <a class="navbar-item" href="/code-of-conduct"> Code of Conduct </a>
+              <a class="navbar-item" href="https://github.com/open-life-science/policies-procedures-and-docs/blob/c342530ffe943bcc1a84a1a8a8e3b5b3c0fa4547/CODE_OF_CONDUCT.md"> Code of Conduct </a>
```
Comment (bebatut) [inline _includes/default-header.html:51]: Could you have the link the main branch there?

File: .github/workflows/update_schedule.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+name:
```
Comment (bebatut) [inline .github/workflows/update_schedule.yml:1]: This file should be not changed. You may have forgotten to go back to the main branch before starting this one

**Review by bebatut — COMMENTED** [review]
Thanks @baimamboukar

File: _includes/default-header.html (hunk 1 above)
Comment (baimamboukar) [reply _includes/default-header.html:51]: What do you mean?
I don't understand....
The link is the link of the code-of-conduct file which is in the **main branch**
[This is it](https://github.com/open-life-science/policies-procedures-and-docs/blob/main/CODE_OF_CONDUCT.md)

Comment (bebatut) [reply _includes/default-header.html:51]: the link you put in the HTML file has a commit id instead of main after `blob` and before `CODE_OF_CONDUCT`

Comment (baimamboukar) [reply _includes/default-header.html:51]: Oh! really?
I'm correcting it now ✊

Comment (baimamboukar) [reply _includes/default-header.html]: @bebatut I already corrected it 🎉
Thanks for reviewing

**Comment by baimamboukar** [comment]
> The checks have failed. Could you check and fix the errors?

I verified in the netlify logs! This is what is causing the issue

image

**Any idea of How to go about it ?**

**Comment by baimamboukar** [comment]
> > The checks have failed. Could you check and fix the errors?
> 
> I verified in the netlify logs! This is what is causing the issue
> 
> image
> 
> **Any idea of How to go about it ?** 

Or maybe I should another pull request and close this one?

**Comment by baimamboukar** [comment]
> The checks have failed. Could you check and fix the errors?

I have fixed the other checks but this one is persisting!

image

**The reason is.... The funders.md file contains an image which has a HTTP link. Having a HTTP *S* link will solve the problem**

### What should I do @bebatut

**Comment by bebatut** [comment]
You still need to remove the `code_of_conduct.md` file but could you change the link in the `_posts/2020-10-01-annual-report-part-1.md`
The other error about the mixed content, I can fix it in another PR

**Comment by baimamboukar** [comment]
> You still need to remove the `code_of_conduct.md` file but could you change the link in the `_posts/2020-10-01-annual-report-part-1.md` The other error about the mixed content, I can fix it in another PR

Which link should I change there ?

This is becoming really consufing!
I can close the two PR and restart from scratch in one Pull Request....
What do you think about this ?

**Comment by baimamboukar** [comment]
When I try to combine the two PRs, into one.... It is prompting me to ``CREATE A PULL REQUEST``
Seems like it is not possible to combine pull requests @bebatut

**Comment by bebatut** [comment]
Yep start again from scratch it will be much cleaner. 
The idea,
- go to the main branch
- update it
- create a new branch
- make all changes related to the code of conduct in the branch
- push to GitHub
- open 1 PR

**Comment by bebatut** [comment]
For the link in the _post file, you need to change it to link to the code of conduct in the Policy GitHub repository, same link as in the navbar

**Comment by baimamboukar** [comment]
> Yep start again from scratch it will be much cleaner. 
> The idea,
> - go to the main branch
> - update it
> - create a new branch
> - make all changes related to the code of conduct in the branch
> - push to GitHub
> - open 1 PR
> 
> 
I am working on it **NOW** 🦸
Thanks very much for your guidance and support @bebatut 
I'm grateful...☄️

**Pull request closed without merging**